### DISCUSSION TITLE
Welcome to dynamic-links Discussions!

### DISCUSSION THREAD

**Discussion by yedincisenol** [yedincisenol/dynamic-links#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by weeliem** [comment]
Hi, how can i set the AndroidInfo and iOSInfo to allow open app instead of browser when clicked the dynamic link?

**Reply by yedincisenol** [reply]
This is what the dynamic links created for. You can read the original project documentation: https://firebase.google.com/docs/dynamic-links

**Reply by weeliem** [reply]
I mean with the library yedincisenol/dynamic-links for laravel. There has no documentation for me to refer how can we add in the androidInfo and iOSInfo.

**Reply by yedincisenol** [reply]
There is an example file, it helps? https://github.com/yedincisenol/dynamic-links/blob/master/example.php#L30

**Reply by weeliem** [reply]
Ah great! DIdn't notice there is an example file right there. Thanks alot :)
